plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -192,7 +192,7 @@ void createPluginDir(boolean failIfExists) {
         try {
             Files.createDirectories(pluginDir.toPath());
         } catch (IOException e) {
-            throw new DirectoryCreationException("Unable to create plugin directory", e);
+            throw new DirectoryCreationException("Unable to create plugin directory: "+"'"+pluginDir+"', supply a directory with -d <your-directory>", e);
```

Comment:
Let's use a format to make it more readable
```suggestion
throw new DirectoryCreationException(String.format("Unable to create plugin directory: '%s', supply a directory with -d <your-directory>", pluginDir), e);
```

Reply:
I was thinking about it but then stayed with concatatenation. This is probably better. I refrained from using it because I thought there might be a typeError when considering pluginDir to be a string. Thanks!

Reply:
you can re-open the pull request and then apply the suggestion directly in GitHub